### PR TITLE
adjust title link and add note

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/3042

### CODE REVIEW

**Comment by get-itips** [comment]
@TroyStarr Can you help me here to understand if we need to remove Subscription Edition from the applies to in this article? Given that classic web apps [are not supported](https://docs.microsoft.com/en-us/sharepoint/what-s-new/new-and-improved-features-in-sharepoint-server-subscription-edition#new-spwebapplication-powershell-cmdlet) and claims are created by default in that edition?

if yes, I'll update this pr, thank you

**Comment by TroyStarr** [comment]
Hi @get-itips, you're right that Subscription Edition doesn't support classic authentication mode.  That said, since this article is primarily about claims authentication mode, I think it's reasonable to say that the article applies to Subscription Edition.  But we can add a note in the section about classic authentication mode to say that it isn't supported with SharePoint Server Subscription Edition.

**Comment by get-itips** [comment]
> Hi @get-itips, you're right that Subscription Edition doesn't support classic authentication mode. That said, since this article is primarily about claims authentication mode, I think it's reasonable to say that the article applies to Subscription Edition. But we can add a note in the section about classic authentication mode to say that it isn't supported with SharePoint Server Subscription Edition.

Great Troy, thanks for taking the time, will do just that

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointServer/security-for-sharepoint-server/create-claims-based-web-applications.md [hunk 1]
```diff
@@ -208,6 +208,9 @@ Use the procedure in this section to create a new claims-based SharePoint Server
 <a name="section3"> </a>
 
 Use the procedure in this section to create a new classic-mode SharePoint Server web application using PowerShell.
+
+   > [!NOTE]
+   > Windows Classic authentication mode is not supported in Subscription Edition, see [New and improved features in SharePoint Server Subscription Edition](../what-s-new/new-and-improved-features-in-sharepoint-server-subscription-edition.md), for more information.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/security-for-sharepoint-server/create-claims-based-web-applications.md:213]:
```suggestion
   > Windows Classic authentication mode is not supported in Subscription Edition. See [New and improved features in SharePoint Server Subscription Edition](../what-s-new/new-and-improved-features-in-sharepoint-server-subscription-edition.md) for more information.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Pull request merged**